ROBOT_FRAME: "base_footprint"
MAX_VELOCITY_X: 2.5
MIN_VELOCITY_X: -2.5
MAX_VELOCITY_Y: 2.5
MIN_VELOCITY_Y: -2.5
MAX_YAWRATE: 0
MAX_D_YAWRATE: 0
MAX_ACCELERATION: 2.0
